src/iOS/Xamarin.iOS/Samples/Network analysis/RouteAroundBarriers/RouteAroundBarriers.cs

```diff
@@ -325,11 +328,11 @@ private void HandleOpenSettings(object sender, EventArgs e)
             controller.NavigationBar.Items[0].SetRightBarButtonItem(closeButton, false);
             // Show the table view in a popover.
             controller.ModalPresentationStyle = UIModalPresentationStyle.Popover;
-            controller.PreferredContentSize = new CGSize(320, 250);
+            controller.PreferredContentSize = new CGSize(450, 250);
```

Comment:
Looking at this again, I think the size issue might be more about the alignment/settings of the stackview inside the popover than the preferred content size.